__tests__/app.test.js

```diff
@@ -372,4 +372,27 @@ describe('GET /api/users', () => {
             })
         })
     });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles?topic', () => {
+    it('200: when queried with topic returns only articles of specified topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            body.forEach((article) => {
+                expect(Object.keys(article).length).toBe(8)
+                expect(article.topic).toBe("mitch");
+            })
+        });
+    });
+    it('when queried with a topic that matches nothing then return 404 error', () => {
+        return request(app)
+        .get('/api/articles?topic=invalid')
+        .expect(404)
+        .then(( {body} ) => {
+            expect(body.msg).toBe("Not Found")
+        })
+    });
+});
```

Comment:
Nearly there...
One test missing:
We have a valid topic which has no associated articles (topic=paper)
In this case we would expect 200 returns an empty array